### PR TITLE
Translate : docs/general/implementation.md

### PR DESCRIPTION
**Pull request by neoseurae12**
Related issue: #153

Let's get it done 🔥

### CODE REVIEW

File: docs/general/implementation.md [hunk 1]
```diff
@@ -183,10 +183,6 @@ HTTP 파이프라인은 여러 정책에 의해 감싸지는 HTTP 전송으로 
 ## 분산 추적
 
 분산 추적 메커니즘은 소비자가 프론트엔드에서 백엔드까지 코드를 추적할 수 있도록 합니다. 분산 추적 라이브러리는 고유한 작업 단위인 스팬(span)을 생성합니다.  각 스팬은 부모-자식 관계에 있습니다. 코드의 계층 구조가 깊어질수록 더 많은 스팬이 생성됩니다. 이후에 이러한 스팬은 필요에 따라 적절한 수신기로 내보내질 수 있습니다. 스팬을 추적하기 위해, _분산 추적 컨텍스트_ (이하 컨텍스트)가 각각의 연속된 레이어에 전달됩니다. 이 항목에 대한 자세한 내용은, 추적에 대한 [OpenTelemetry] 항목을 참조하세요.
```
Comment (neoseurae12) [inline docs/general/implementation.md:185]: "These spans can then be exported to a suitable receiver as needed."에서,
여기서 'receiver'(수신기)란 무엇인가? 그 예시에 어떤 것들이 있는가?

**Comment by neoseurae12** [comment]
Now you can see 'General Guidelines: Implementation' ALL in Korean. 😁

'General Guidelines: Implementation'의 번역을 완료했습니다. 이제 모두 한국어로 읽어보실 수 있습니다 😁

File: docs/general/implementation.md (hunk 1 above)
Comment (ianychoi) [reply docs/general/implementation.md:185]: I think you can find more details on distributed tracing and receivers on https://learn.microsoft.com/ko-kr/dotnet/core/diagnostics/distributed-tracing-concepts

File: _data/sidebars/general_sidebar.yml [hunk 2]
```diff
@@ -33,7 +33,7 @@ entries:
         url: /general_terminology.html
       - title: API 디자인
         url: /general_design.html
-      - title: Implementation (영문)
+      - title: 구현 (번역 중)
```
Comment (ianychoi) [inline _data/sidebars/general_sidebar.yml:36]: Would you remove "(번역 중)" since you mention that you finished translating this part? Thanks!

**Review by ianychoi — CHANGES_REQUESTED** [review]

File: _data/sidebars/general_sidebar.yml (hunk 2 above)
Comment (neoseurae12) [reply _data/sidebars/general_sidebar.yml]: Oh thanks for your reviewing. can you check this out once again? Thanks :)

**Pull request merged**